src/store/middlewares/peerMiddleware.tsx

```diff
@@ -35,6 +35,8 @@ const createPeerMiddleware = ({
 									roles,
 									raisedHand,
 									raisedHandTimestamp,
+									escapeMeeting,
```

Comment:
Remove all knowledge about the escape state of other peers, only the server has to know.

Reply:
Done